WordPress/Classes/ViewRelated/Domains/Views/SiteDomainsView.swift

```diff
@@ -2,7 +2,8 @@ import SwiftUI
 import WordPressKit
 
 /// The Domains dashboard screen, accessible from My Site
-struct DomainsDashboardView: View {
+struct SiteDomainsView: View {
```

Comment:
Thanks for renaming this! I kept confusing it as a "Dashboard" related component.